.setup/INSTALL_SUBMITTY_HELPER_BIN.sh

```diff
@@ -70,6 +70,10 @@ chmod 550 ${SUBMITTY_INSTALL_DIR}/sbin/authentication.py
 # everyone needs to be able to run this script
 chmod 555 ${SUBMITTY_INSTALL_DIR}/sbin/killall.py
 
+# DAEMON_USER should be able to run these scripts as root
+echo "%${DAEMON_GROUP} ALL = (root) NOPASSWD: ${SUBMITTY_INSTALL_DIR}/sbin/create_course.sh" >> /etc/sudoers
```

Comment:
won't this result in duplicate lines when this script is re-run?

Reply:
Yeah, it will. We really shouldn't be trying to modify this file directly but rather add a file under `/etc/sudoers.d/` (much like how we handle cron).

Reply:
Thanks for pointing this out. Is there any example that could be my reference? (I didn't find `sudoers.d` in the code) @MasterOdin

Reply:
It's not something we're currently doing. Right now there's also two places in install_system.sh that directly modifies `/etc/sudoers`.

I think for now, please just create a file `.setup/submitty_sudoers` with the above lines you were echoing into `/etc/sudoers` and then in `.setup/INSTALL_SUBMITTY.sh`, where we install the crontab file, add:
```
cat ${SUBMITTY_REPOSITORY}/.setup/submitty_sudoers | envsubst > /etc/sudoers.d/submitty
chmod 0440 /etc/sudoers.d/submitty
chown root:root /etc/sudoers.d/submitty
```

Reply:
Looks like the env var is not substituted. I also found it hard-coded in `submitty_crontab`. Is it generated somewhere else?

https://github.com/Submitty/Submitty/blob/e51aad995171d7b4257c1acf079c5ce1da335bcf/.setup/submitty_crontab#L14

Reply:
Yeah, it's because `envsubst` only uses environment variables which in bash means that the variables have `export` in them, and it won't use variables defined in the current script. Don't worry about using the variables here, I'll handle it in a future PR as part of the larger #4011 refactors.